src/audio/src/src.c

```diff
@@ -987,6 +987,12 @@ static int src_init(struct processing_module *mod)
 
 	comp_dbg(dev, "src_init()");
 
+	if (!dev->ipc_config.type != SOF_COMP_SRC || !cfg->init_data ||
```

Comment:
yes, just please remove the stray `!` before `dev` and it should be good!

Reply:
Oops, guess there is no mock-up test for src, since run-mocks.sh did not catch that.